news/_posts/core-weekly/2019-05-31-coreweekly-week-21-2019.md

```diff
@@ -0,0 +1,147 @@
+---
+layout: post
+title:  "PrestaShop Core Weekly - Week 21 of 2019"
+subtitle: "An inside look at the PrestaShop codebase"
+date:   2019-05-31 10:30:00
+authors: [ AntoineThomas ]
+icon: icon-calendar
+tags:
+ - core-weekly
+---
+
+This edition of the Core Weekly report highlights changes in PrestaShop's core codebase from Monday 20th to Sunday 26th of May 2019.
+
+![Core Weekly banner](/assets/images/2018/12/banner-core-weekly.jpg)
+
+
+## General messages
+
+Dear Developers,
+
+As you will see, the work on the upcoming 1.7.6 version is going on well, and the faceted search module is still needing some improvements.
+
+The PrestaShop Developers Team is also working on content for the [PSD 2019](https://www.prestashop.com/fr/evenements/prestashop-day-paris): most of them have a talk or a workshop. And they are eager to meet your there.
```

Comment:
```suggestion
The PrestaShop Developers Team is also working on content for the [PSD Paris 2019](https://www.prestashop.com/fr/evenements/prestashop-day-paris): most of them have a talk or a workshop. And they are eager to meet your there.
```

Reply:
```suggestion
The PrestaShop Developers Team is also working on content for the upcoing [PSD 2019 Event in Paris](https://www.prestashop.com/fr/evenements/prestashop-day-paris): most of them have a talk or a workshop. And they are eager to meet your there.
```

I don't think most people are aware what PSD is.

Reply:
"the" or "our" event ?